src/main/java/com/EighthLight/app/Ui.java

```diff
@@ -28,42 +28,44 @@ public String getInput() {
         }
     }
 
+    public String makeDivider(String rowString) {
```

Comment:
some awesome abstractions in here! Tons of progress since the last UI I saw 🚀 🚀 🚀